src/components/Users/Users.js

```diff
@@ -1,40 +1,36 @@
 import { Breadcrumb } from 'react-breadcrumbs';
 import { connect } from 'react-redux';
+import { formatStatus } from './UsersUtils';
 import { invitationCreate, invitationsLoad } from 'actions/invitationActions';
 import { push } from 'connected-react-router';
-import { relativeDate } from 'lib/helpers.js';
-import { spinner } from 'images';
 import {
   userDelete,
   userRemoveExpiration,
   usersLoad,
 } from 'actions/userActions';
-import _ from 'underscore';
-import BootstrapModal from 'react-bootstrap/lib/Modal';
-import BootstrapTable from 'react-bootstrap-table-next';
 import Button from 'UI/Button';
-import copy from 'copy-to-clipboard';
+import DeleteUserModal from './DeleteUserModal';
```

Comment:
I like these refactorings, they make the code better! Really appreciate it.

Would've been best in a separate PR though to keep this one focused on the topic of creating that hook.